frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx

```diff
@@ -0,0 +1,102 @@
+import * as _ from 'lodash';
+import { Humanize } from '@console/internal/components/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import { Colors } from './bar-colors';
+
+const getTotal = (stats: StackDataPoint[]) => {
+  return stats.reduce((total, dataPoint) => total + dataPoint.y, 0);
+};
+
+const addOthers = (stats: StackDataPoint[], totalUsed: string, formatValue: Humanize) => {
+  const top5Total = getTotal(stats);
+  const others = Number(totalUsed) - top5Total;
+  const othersData = {
+    x: '',
+    y: others,
+    name: 'Others',
+    color: Colors.OTHER,
+    label: formatValue(others).string,
+  };
+  return othersData;
+};
+
+export const addAvailable = (
+  stats: StackDataPoint[],
+  total: string,
+  used: string,
+  totalUsed: string,
+  formatValue: Humanize,
+) => {
+  const availableInBytes = Number(total) - Number(used);
+  let othersData = {};
+  if (stats.length === 5) {
+    othersData = addOthers(stats, totalUsed, formatValue);
+  }
+  const availableData = {
+    x: '',
+    y: availableInBytes,
+    label: `${formatValue(availableInBytes).string}`,
```

Comment:
```suggestion
    label: formatValue(availableInBytes).string,
```
@a2batic I think this is what @gnehapk meant here.

Reply:
ack